test/fixtures/files/example-01-file.smart-health-card

```diff
@@ -0,0 +1 @@
+{"verifiableCredential":["eyJ6aXAiOiJERUYiLCJhbGciOiJFUzI1NiIsImtpZCI6Il8xNHBlcVFQaXhXX2taUkVINE9KMGxlUkNsTURjTVlocHd2TVhDY1JBRncifQ.hVJLb5wwEP4v0ysPQ3cTlmPSSw9RKzXtpdqDMcPiyg9kG9RtxH_vGJLsKuluuSDjb74X8wTSe6ihD2Go81xZwVVvfag_MsYgAdN0UBc3ZXVT7HZVkcAkoH6CcBwQ6p_LmKc5r7kLPXIV-kxw1_oP6yGNB6K5jBN2km2xu4qRWo9G_uFBWnMVqHjj0I8qwD4B4bBFEyRX38bmF4oQjXe9dD_Q-chUwyZjWUGM8evdaFqFp3AgrFI0tWoSkTtSYmIYlfruFAFIyo5OYB2LejlEAsM1rliupaIxeJDE5Qh2kBOa2NwDF7CfyWYjKcQnHqIkdVykbJuy6ozxcbXzleKTC5jn5J8mijcmfOBh9EsOPSgMGP_DxIWQBu9tu2CEbaU5LFb90QfUp03o1W1m3SGP1eRetrmYfhOBWCahLEqY93MCw7MrInPYoUMT1c-rIZAVYnTLVcz5KPVCwUrKWqVs8z7r5_MffilweSlwJw1Xr16vplRWGhFznqJt2XZTpbdrPOy6uALTReMTV-NSJ28UvSniEP6j6Y3V2GbSdDb3tJWvygWryooR7SL9ppIvjUc3vTSyf3f_vL4zPX8B.d_wXh1960tnhE0OG1NZHc54EjO3e-fuKDNTin0JUd2TzzRdAgCvc7KDx7dvhlFomWXphjuxBdH6D8j8I2w2H9g"]}
```

Comment:
Is this ever used anywhere?

Reply:
nope - removed it

Also for the valueSetTest - it is covered through the lab_results_helper_test. But maybe in the future if we add more functionality, a test might be needed so left it as is (similar to vaccine_test)

Reply:
Sounds good, that's kind of what I figured.